src/App.js

```diff
@@ -35,22 +35,30 @@ function App() {
     departureLocation,
     arrivalLocation
   ) {
-    setSavedFlightPlanData([
-      ...savedFlightPlanData,
-      {
-        id,
-        departureAirport,
-        departureRunway,
-        arrivalAirport,
-        arrivalRunway,
-        flightDate,
-        flightTime,
-        aircraft,
-        numberOfPassengers,
-        departureLocation,
-        arrivalLocation,
-      },
-    ]);
+    if (savedFlightPlanData.find((item) => id === item.id)) {
+      const myId = savedFlightPlanData.findIndex((item) => item.id === id);
+      console.log(myId);
```

Comment:
I don't think you need the console.log here :)

Reply:
This was for testing! Removed! Thanks